src/content/configuration/output.md

```diff
@@ -356,86 +350,111 @@ MySubModule.doSomething();
+//your users will use your library like the following:
 window.MyLibrary.doSomething();
  ```
 
 
-`libraryTarget: "global"` - When your library is loaded, **the return value of your entry point** will be part `global` object.
+`libraryTarget: "global"` - When your library is loaded, **the return value of your entry point** will be part `global` object under the property named by `output.library`.
 
  ```javascript
  global["MyLibrary"] = _entry_return_;
 
-//your users will use your library like:
+//your users will use your library like the following:
 global.MyLibrary.doSomething();
  ```
 
 
-`libraryTarget: "commonjs"` - When your library is loaded, **the return value of your entry point** will be part of the exports object. As the name implies, this is used in CommonJS environments:
+`libraryTarget: "commonjs"` - When your library is loaded, **the return value of your entry point** will be part of the `exports` object under the property named by `output.library`. As the name implies, this is used in CommonJS environments:
 
 ```javascript
 exports["MyLibrary"] = _entry_return_;
 
-//your users will use your library like:
+//your users will use your library like the following:
 require("MyLibrary").doSomething();
 ```
 
+### Exposing the library under formalized module definition systems
+
+These options will result in a bundle that comes with a more complete header to ensures compatibility with various module systems.  The `output.library` option will take on a different meaning under the following `output.libraryTarget` options.
+
 
 `libraryTarget: "commonjs2"` - When your library is loaded, **the return value of your entry point** will be part of the exports object. As the name implies, this is used in CommonJS environments:
 
 ```javascript
 module.exports = _entry_return_;
 
-//your users will use your library like:
+//your users will use your library like the following:
 require("MyLibrary").doSomething();
 ```
 
-T> Wondering the difference between CommonJS and CommonJS2? Check [this](https://github.com/webpack/webpack/issues/1114) out (they are pretty much the same).
-
+Note that `output.library` is omitted, thus it is not required for this particular `output.libraryType`.
 
-`libraryTarget: "amd"` - In this case webpack will make your library an AMD module.
+T> Wondering the difference between CommonJS and CommonJS2 is? While they are similar, there are some subtle differences between them that are not usually relevant in the context of webpack. (For further details, please [read this issue](https://github.com/webpack/webpack/issues/1114).)
 
-Note that your entry chunk must be defined with the `define` property, if not, webpack will create the AMD module, but without dependencies. The output will be something like this:
 
-```javascript
-define([], function() {
-  // This module returns is what your entry chunk returns
-});
-```
+`libraryTarget: "amd"` - In this case webpack will make your library an AMD module.
```

Comment:
"In this case webpack will make your library an AMD module." => "This will expose your library as an AMD module."